### PR TITLE
Rewrite of sign in section of troubleshooting guide

### PR DESCRIPTION
**Pull request by neoarz**

### CODE REVIEW

**Comment by polymo1** [comment]
WIP or ready for review?

**Comment by neoarz** [comment]
> WIP or ready for review?

ready for review

File: docs/troubleshooting/troubleshooting.mdx [hunk 1]
```diff
@@ -25,18 +25,47 @@ description: A guide to troubleshooting SideStore.
...
+#### Sidestore Freezing or Displaying an Error Code During Sign-In
+
+If you encounter freezing or error codes while attempting to sign in to Sidestore, there could be several potential causes. Follow these steps to resolve the issue:
+
+1. **Change the Anisette Server:**
+   The most common solution is to switch to a different Anisette server. Anisette servers may occasionally experience downtime. Switching to an alternative server typically resolves the issue. Do this:
+   - Open Sidestore settings.
+   - Scroll down to the "Anisette Server" option.
+   - Select a different server from the list.
+   - We advise you to try the "Macley" server first before the others, as it is one of the most stable.
```
Comment (polymo1) [inline docs/troubleshooting/troubleshooting.mdx:39]: No, we are actively working to fix the server problems, but we should not say a specific server to use. Say to use a different server

**Review by polymo1 — CHANGES_REQUESTED** [review]

File: docs/troubleshooting/troubleshooting.mdx (hunk 1 above)
Comment (neoarz) [reply docs/troubleshooting/troubleshooting.mdx]: fixed

**Review by polymo1 — APPROVED** [review]
Lgtm

**Pull request merged**